packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin.spec

```diff
@@ -0,0 +1,46 @@
+%global goproject github.com/NVIDIA
+%global gorepo k8s-device-plugin
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 0.10.0
+%global rpmver %{gover}
+
+Name: %{_cross_os}nvidia-k8s-device-plugin
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary: K8s device plugin for NVIDIA GPUs
+License: Apache-2.0
+URL: https://github.com/NVIDIA/k8s-device-plugin
+Source0: https://%{goimport}/archive/v%{gover}/v%{gover}.tar.gz
+Source1: nvidia-k8s-device-plugin.service
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n %{gorepo}-%{gover} -p1
+%cross_go_setup %{gorepo}-%{gover} %{goproject} %{goimport}
+# We don't set `-Wl,-z,now`, because the binary uses lazy loading
+# to load the NVIDIA libraries in the host
+%global set_env \
+%cross_go_configure %{goimport} \\\
+export CGO_LDFLAGS="-Wl,-z,relro" \\\
+%{nil}
+
+%build
+%set_env
```

Comment:
nit: this is fine but I'd probably do it like this just so `%build` looks more like the other Go packages -

```suggestion
%build
%cross_go_configure %{goimport}
# We don't set `-Wl,-z,now`, because the binary uses lazy loading
# to load the NVIDIA libraries in the host
export CGO_LDFLAGS="-Wl,-z,relro"
```